### ISSUE TITLE
CPU/Memory Benchmark of Ebiten

### ISSUE THREAD

**Issue by wasedaigo** [hajimehoshi/ebiten#902]
Currently there is no good way to know if ebiten is a better choice compared with other major game engines when it comes to the performance. It'd be great to have a benchmark comparison.

**Comment by hajimehoshi** [comment]
Good point. My gut feeling is that it seems hard to create a fair comparison since the number of features are different.

**Comment by wasedaigo** [comment]
My expectation is ebiten out performs others as it has limited feature set.
